react-native-libraries.json

```diff
@@ -2083,40 +2083,61 @@
       "https://github.com/invertase/react-native-google-mobile-ads/tree/main/RNGoogleMobileAdsExample"
     ],
     "ios": true,
-    "android": true
+    "android": true,
+    "newArchitecture": true
   },
   {
     "githubUrl": "https://github.com/invertase/react-native-firebase/tree/main/packages/analytics",
     "npmPkg": "@react-native-firebase/analytics",
     "ios": true,
+    "tvos": true,
+    "android": true,
+    "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/invertase/react-native-firebase/tree/main/packages/app-check",
+    "npmPkg": "@react-native-firebase/app-check",
+    "ios": true,
+    "tvos": true,
+    "android": true,
+    "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/invertase/react-native-firebase/tree/main/packages/app-distribution",
+    "npmPkg": "@react-native-firebase/app-distribution",
+    "ios": true,
```

Comment:
Hey @mikehardy, thanks for the updates! 👍 

Let's move new libraries at the end of file:
* https://github.com/react-native-community/directory?tab=readme-ov-file#how-do-i-add-a-library

Reply:
May I please please request that they go here, since they are a monorepo of packages and I added them up with their siblings in alphabetical order? I see that the order is for "recently added" but having these packages appear as "fresh" (hey! recently added! type information for users) honestly has no value here in this case for us if that's the only reason.

If there is no way to bend on this then...okay

Reply:
The new status means new to the directory, not fresh packages. Moving them down could help with discoverability, but it's fine if you prefer having them grouped.

Reply:
Thanks! You inferred correctly in case my statement was ambiguous - there was no value in discoverability really, if people are using firebase they either know these exist already or they will just know they need them when they need them and they'll expect them to be there and they will be :-). Firebase and all its little services are like that